{{cookiecutter.project_name}}/.travis.yml

```diff
@@ -1,23 +1,29 @@
 language: python
 
+env:
+  - DOCKER_COMPOSE_VERSION=1.21.0
+
 python:
   - "3.6"
 
 services:
-  - postgresql
+  - docker
 
-cache:
-  - pip
+before_install:
+  # install newer compose version
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
 
-install:
-  - echo "ENV=travis" > .env
-  - make install-dev
+  # Let's stop postgresql
+  - sudo service postgresql stop
```

Comment:
Why is there a postgresql service around in the first place? Is this really required?

Reply:
https://github.com/travis-ci/travis-ci/issues/4842

Reply:
Fair point. Added link to issue as comment.